portfolio/src/main/webapp/index.html

```diff
@@ -8,77 +8,81 @@
-        </br>
-        <h1 class="name">Liyah Coleman</h1>
-        </br>
-
-        <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
-            <ol class="carousel-indicators">
-                <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
-                <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
-            </ol>
-            <div class="carousel-inner" >
-                <div class="carousel-item active">
-                <img src="/images/one.jpg" class="d-block w-100" style="height:800px" style="width:800px">
-                </div>
-                <div class="carousel-item">
-                <img src="/images/two.jpg" class="d-block w-100" style="height:800px" style="width:800px">
+        <div id="content"> 
+            </br>
+            <h1 class="name">Liyah Coleman</h1>
+            </br>
+            <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+                <ol class="carousel-indicators">
+                    <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+                    <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+                </ol>
+                <div class="carousel-inner" >
+                    <div class="carousel-item active">
+                    <img src="/images/one.jpg" class="d-block w-100" style="height:550px">
+                    </div>
+                    <div class="carousel-item">
+                    <img src="/images/two.jpg" class="d-block w-100" style="height:550px"> 
+                    </div>
                 </div>
             </div>
-        </div>
-        </br>
+            </br> 
 
-        <nav id="navbar-example2" class="navbar navbar-light bg-light">
-            <a class="navbar-brand" href="#"></a>
-            <ul class="nav nav-pills">
-                <li class="nav-item"> 
-                    <a class="nav-link" href="#one">Projects</a>
+            <ul class="nav nav-pills nav-fill mb-3" id="pills-tab" role="tablist">
+                <li class="nav-item" role="presentation">
+                    <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-selected="true">About Me</a>
+                </li>
+                <li class="nav-item" role="presentation">
+                    <a class="nav-link" id="pills-profile-tab" data-toggle="pill" href="#pills-profile" role="tab" aria-controls="pills-profile" aria-selected="false">Projects</a>
                 </li>
-                <li class="nav-item">
-                    <a class="nav-link" href="#two"> Updates </a>
+                <li class="nav-item" role="presentation">
+                    <a class="nav-link" id="pills-contact-tab" data-toggle="pill" href="#pills-contact" role="tab" aria-controls="pills-contact" aria-selected="false">Updates</a>
```

Comment:
this line is pretty long, you could break it up over multiple lines like so:

https://stackoverflow.com/questions/44944936/indenting-html-tags-on-multiple-lines